src/app/components/Validator/components/ValidatorDuties.tsx

```diff
@@ -78,42 +64,12 @@ export default (props: ValidatorDutiesProps) => {
   };
 
   const getRenderedSigners = (operators: any[], missingOperators: any[]) => {
+    const operatorList = [...operators, ...missingOperators];
     return (
       <>
-        {operators.length ? (
-          <SuccessChip
-            className={chipClasses.chip}
-            label={operators.map((o, oi) => (
-              <ChipLink
-                key={`operators-success-${oi}`}
-                className={classes.Link}
-                href={`${config.routes.OPERATORS.HOME}/${o.id}`}
-                style={{ maxWidth: 100 }}
-              >
-                <Typography noWrap style={{ fontSize: 14 }}>{o.name}</Typography>
-              </ChipLink>
-            ))
-            }
-            onDelete={() => {}}
-            deleteIcon={<CheckCircleIcon />}
-          />
-        ) : ''}
-        {missingOperators.length ? (
-          <FailureChip
-            className={chipClasses.chip}
-            label={missingOperators.map((o, oi) => (
-              <ChipLink
-                key={`operators-failed-${oi}`}
-                className={classes.Link}
-                href={`${config.routes.OPERATORS.HOME}/${o.id}`}
-                style={{ maxWidth: 100 }}
-              >
-                <Typography noWrap style={{ fontSize: 14 }}>{o.name}</Typography>
-              </ChipLink>
-            ))}
-            onDelete={() => {}}
-          />
-        ) : ''}
+        <Grid className={classes.OperatorConsensusWrapper}>
+          {operatorList.map((operator: any, index: number) => (<OperatorConsensusSlot key={operator.id} operatorName={operator.name} operatorId={operator.id} status={operator.status} isNotFirst={index !== 0} isNotLast={index !== operatorList.length - 1} />))}
```

Comment:
This line is too long, do something with it to be better readable and fit into 80 chars width

Reply:
done